test/import_package_uri_test.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:sass/src/exception.dart';
+import 'package:test/test.dart';
+import 'package:sass/sass.dart';
+
+main() {
+  group('import "package:" uri', () {
+    test('found', () async {
+      var result = await render('test/styles/import-package.scss');
```

Comment:
Rather than using assets on the filesystem, Dart Sass tests use `scheduled_test`'s `descriptor` library to create assets in the test itself. This helps keep the test self-contained, and makes diffs a lot easier to read. See `test/cli_shared.dart` for examples.